pytorch_lightning/overrides/fairscale.py

```diff
@@ -29,3 +29,22 @@ def unwrap_lightning_module_sharded(wrapped_model) -> LightningModule:
             model = model.module
 
         return unwrap_lightning_module(model)
+
+
+LightningFullShardedDataParallel = None
+if _FAIRSCALE_FULL_SHARDED_AVAILABLE:
+    from fairscale.nn import FlattenParamsWrapper
+    from fairscale.nn.data_parallel import FullyShardedDataParallel
+
+    class LightningFullShardedDataParallel(_LightningModuleWrapperBase):
+        # Just do this for later docstrings
+        pass
+
+    def unwrap_lightning_module_full_sharded(wrapped_model) -> LightningModule:
```

Comment:
maybe import in this function? At least this function should also live in the global file namespace. Otherwise we may have issues with docs etc.

Reply:
Not sure what you mean here, copied the Sharded DDP version you made

Reply:
Yeah, What I mean is  that the fairscale imports are only needed in this function (at least in this file), so maybe move the imports in there to always have the function in global namespace and not only if SHarded is available?